src/Microsoft.AspNet.Mvc.Core/DefaultControllerActivator.cs

```diff
@@ -3,106 +3,27 @@
 
 using System;
 using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Reflection;
-using Microsoft.AspNet.Http;
-using Microsoft.AspNet.Mvc.Core;
-using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.DependencyInjection;
 
 namespace Microsoft.AspNet.Mvc
 {
+    using CreateControllerThunk = Func<IServiceProvider, object[], object>;
```

Comment:
I was hoping this might commonize that once https://github.com/aspnet/Mvc/issues/1416 gets addressed, cc @kirthik